frontend/app/views/bundle/thankYou.scala.html

```diff
@@ -20,7 +21,7 @@
         <p>To make a membership and subscription offer that fits you, we need your input. <strong>Share your feedback below:</strong></p>
 
         <div class="bundle-thankyou__email-form  bundle-thankyou-js">
-            <script type="text/javascript" src="https://example-memberhsip.formstack.com/forms/js.php/membership?bundle_variant=@bundleVariant.testId"></script><noscript><a href="https://example-memberhsip.formstack.com/forms/membership?bundle-variant=@bundleVariant.testId" title="Online Form">Online Form - Membership</a></noscript><div style="text-align:right; font-size:x-small;"><a href="http://www.formstack.com?utm_source=jsembed&utm_medium=product&utm_campaign=product+branding&fa=h,2576843" title="HTML Form Builder">HTML Form Builder</a></div>
+            <script type="text/javascript" src="https://example-memberhsip.formstack.com/forms/js.php/membership?bundle_variant=@bundleVariant.testId&selected_option=@selectedOption"></script><noscript><a href="https://example-memberhsip.formstack.com/forms/membership?bundle-variant=@bundleVariant.testId" title="Online Form">Online Form - Membership</a></noscript><div style="text-align:right; font-size:x-small;"><a href="http://www.formstack.com?utm_source=jsembed&utm_medium=product&utm_campaign=product+branding&fa=h,2576843" title="HTML Form Builder">HTML Form Builder</a></div>
```

Comment:
You are going to need to get in contact with central.production (eg Calvin Dickson) to set up the proper form in the official membership formstack account - probably a good idea to get in contact with them now to start building the form.